### PR TITLE
[ContextMenu] Add submenu int tests

### PR DESCRIPTION
**Pull request by andy-hook**
Sibling PR to https://github.com/radix-ui/primitives/pull/744 that tests the same submenu behaviour but for `ContextMenu`.

### CODE REVIEW

File: cypress/integration/ContextMenu.spec.ts [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+      it('should open submenu and not focus first item when moving pointer over trigger', () => {
+        pointerOver('Bookmarks →');
+        cy.findByText('Inbox').should('not.be.focused');
+      });
+
+      it('should open submenu and not focus first item when moving pointer over trigger', () => {
+        pointerOver('Bookmarks →');
+        cy.findByText('Inbox').should('not.be.focused');
+      });
```
Comment (jjenzz) [inline cypress/integration/ContextMenu.spec.ts:22]: Duplicate?

Comment (andy-hook) [reply cypress/integration/ContextMenu.spec.ts]: Wups

File: cypress/integration/ContextMenu.spec.ts [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+      it('should close only the focused submenu when pressing left arrow key', () => {
+        cy.findByText('Bookmarks →').type('{enter}');
+        cy.findByText('Modulz →').type('{enter}');
+        cy.findByText('Stitches').type('{leftarrow}').should('not.exist');
+        cy.findByText('Modulz →').should('be.visible');
+        cy.findByText('New Window').should('be.visible');
+      });
+
+      it.only('should close all menus when pressing escape, enter or space key on any item', () => {
```
Comment (benoitgrelard) [inline cypress/integration/ContextMenu.spec.ts:154]: `it.only` means the other tests aren't running right?

Comment (andy-hook) [reply cypress/integration/ContextMenu.spec.ts]: Double whoops, good spot!

File: cypress/integration/ContextMenu.spec.ts [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+    beforeEach(() => {
+      cy.visitStory('contextmenu--submenus');
+      cy.findByText('Right Click Here').rightclick();
+    });
+
+    describe('when using pointer', () => {
+      it('should reposition open menu when right clicking trigger', () => {
+        cy.findByText('Right Click Here').rightclick('topLeft', { force: true });
+        cy.findByText('New Tab').should('be.visible');
+      });
```
[the remark this answers is not in the document]
Comment (andy-hook) [reply cypress/integration/ContextMenu.spec.ts]: Agreed. I've adjusted it to better describe the intent of the test https://github.com/radix-ui/primitives/pull/782/commits/f02556895f5643c66a30e3a409ecc59a8ecff0d4

**Review by jjenzz — APPROVED** [review]

**Pull request merged**